lightning-invoice/src/payment.rs

```diff
@@ -223,6 +224,45 @@ impl<T: Time> PaymentInfo<T> {
 		}
 	}
 }
+
+/// Used to store information about all the HTLCs that are inflight across all payment attempts
+struct AccountForInFlightHtlcs<'a, S: Score> {
```

Comment:
Forgot that this needs to implement `Writeable` for bindings release. See CI failures.

Reply:
FWIW, it doesn't actually need to write anything though. May make sense to move that trait bound to `BackgroundProcessor` in a follow-up once scoring is moved into `DefaultRouter`.

Reply:
when you say it doesn't need to write anything, does that mean just returning `Ok(())` on `write`?

Reply:
Yep